package NCommonServerProto;

message TFileAttachment {
  optional bytes Data = 1;
  optional string MimeType = 2;
  optional string FileName = 3;
}
